src/containers/AdminAutosending/components/AutosendingBasicTargetRow.tsx

```diff
@@ -0,0 +1,103 @@
+import Button from "@material-ui/core/Button";
+import Chip from "@material-ui/core/Chip";
+import green from "@material-ui/core/colors/green";
+import grey from "@material-ui/core/colors/grey";
+import red from "@material-ui/core/colors/red";
+import { makeStyles } from "@material-ui/core/styles";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import MoreIcon from "@material-ui/icons/ArrowForward";
+import PauseIcon from "@material-ui/icons/Pause";
+import PlayIcon from "@material-ui/icons/PlayArrow";
+import { AutosendingTargetFragment } from "@spoke/spoke-codegen";
+import React from "react";
+import { Link } from "react-router-dom";
+
+const useStyles = makeStyles({
+  unstarted: {
+    backgroundColor: grey[200]
```

Comment:
Required: Use theme palette for these colors. If it doesn't exist in theme, create them!